extensions/most-used-content/www/styles.css

```diff
@@ -0,0 +1,62 @@
+/* Styles for the table */
+.content-tbl {
+  font-size: 0.875rem;
+  line-height: 1.125rem;
+}
+
+.content-tbl a {
+  color: inherit;
+  text-decoration: none;
+}
+
+.content-tbl a:hover,
+.content-tbl a:focus {
+  text-decoration: underline;
+  text-decoration-thickness: max(1px, 0.0625rem);
+}
+
+/* Styles for the column headers */
+.header {
+  border-bottom: 2px solid #555;
+  font-size: 0.8125rem;
+  font-weight: 400;
+  text-transform: uppercase;
+}
+
+.header:hover {
+  background-color: #eee;
+}
+
+
+/* Bar chart styles */
+
+.bar-cell {
+  display: flex;
+  align-items: center;
+  width: 100%;
```

Comment:
I believe this is unnecessary since your `bar-chart` flex items have `flex-grow: 1` and will automatically take up all of the horizontal space they can.